internal/cmd/ansible-operator/run/cmd.go

```diff
@@ -294,19 +296,21 @@ func run(cmd *cobra.Command, f *flags.Flags) {
 // if any of those fields are not their default values.
 func exitIfUnsupported(options manager.Options) {
 	var keys []string
-	// The below options are webhook-specific, which is not supported by ansible.
-	if options.CertDir != "" {
-		keys = append(keys, "certDir")
-	}
-	if options.Host != "" {
-		keys = append(keys, "host")
-	}
-	if options.Port != 0 {
-		keys = append(keys, "port")
-	}
 
-	if len(keys) > 0 {
-		log.Error(fmt.Errorf("%s set in manager options", strings.Join(keys, ", ")), "unsupported fields")
+	if options.WebhookServer != nil {
+		// The below options are webhook-specific, which is not supported by ansible.
+		// Adding logs only for the previously supported values through manager.
+		if options.WebhookServer.(*webhook.DefaultServer).Options.CertDir != "" {
+			keys = append(keys, "certDir")
+		}
+		if options.WebhookServer.(*webhook.DefaultServer).Options.Host != "" {
+			keys = append(keys, "host")
+		}
+		if options.WebhookServer.(*webhook.DefaultServer).Options.Port != 0 {
+			keys = append(keys, "port")
+		}
```

Comment:
If the only way to configure webhook options now are via the `options.WebhookServer` field, then removing these specific field checks and logging that webhook options are not supported and erroring out should be sufficient

Reply:
`option.WebhookServer` now supports many other options apart from these three. But the existing ansible owners only know of 3 configurable values CertDir, Host, Port. Which is why this explicitly checks if these 3 are set, and if so, logs specifically that these are not acceptable. 

This check can be removed and can be reduced to verifying just `options.WebhookServer`. But it can be a TODO in next release. This logging is to maintain compatibility between what is existing.